test/integration/test_runner.py

```diff
@@ -14,6 +14,7 @@
 from test.utils.common import iterate_timeout
 
 
+@pytest.mark.xfail(reason='Test is unstable')
```

Comment:
If it's flaky, it should be
```suggestion
@pytest.mark.xfail(reason='Test is unstable', strict=False)
```